WooCommerce/Classes/ViewRelated/Products/Media/MediaPickingCoordinator.swift

```diff
@@ -11,13 +11,19 @@ final class MediaPickingCoordinator {
         return DeviceMediaLibraryPicker(onCompletion: onDeviceMediaLibraryPickerCompletion)
     }()
 
+    private let siteID: Int64
     private let onCameraCaptureCompletion: CameraCaptureCoordinator.Completion
     private let onDeviceMediaLibraryPickerCompletion: DeviceMediaLibraryPicker.Completion
+    private let onWPMediaPickerCompletion: WordPressMediaLibraryImagePickerViewController.Completion
 
-    init(onCameraCaptureCompletion: @escaping CameraCaptureCoordinator.Completion,
-         onDeviceMediaLibraryPickerCompletion: @escaping DeviceMediaLibraryPicker.Completion) {
+    init(siteID: Int64,
+         onCameraCaptureCompletion: @escaping CameraCaptureCoordinator.Completion,
+         onDeviceMediaLibraryPickerCompletion: @escaping DeviceMediaLibraryPicker.Completion,
+         onWPMediaPickerCompletion: @escaping WordPressMediaLibraryImagePickerViewController.Completion) {
```

Comment:
camera capture and device media library could share the same completion block, but WP media picker has a different response since the WP media already exist remotely. maybe we can consolidate the completion block types next time we add an image source?

Reply:
Yeah, this is fine for now. If the code grows, we can adjust. 🙂